hazelcast-spring/src/main/java/com/hazelcast/spring/HazelcastConfigBeanDefinitionParser.java

```diff
@@ -1733,6 +1740,80 @@ private void handleSecurity(Node node) {
             configBuilder.addPropertyValue("securityConfig", beanDefinition);
         }
 
+        private void handleRealms(Node node, BeanDefinitionBuilder securityConfigBuilder) {
+            ManagedMap<String, BeanDefinition> realms = new ManagedMap<>();
+            for (Node child : childElements(node)) {
+                String nodeName = cleanNodeName(child);
+                if ("realm".equals(nodeName)) {
+                    realms.put(getAttribute(child, "name"), handleRealm(child, securityConfigBuilder));
```

Comment:
Wouldn't a bean reference be more Spring-y? (Not sure if it makes sense here, just asking because we seem to do that where possible.)

Reply:
I would leave it this way, as it's not a top-level configuration, but part of the security configuration.